PowerShell not working without eval flag

### Code of Conduct

- [x] I agree to follow this project's Code of Conduct

### What happened?

Since version `27.1.1` the previously working init line stopped working properly, reverts back to the default theme.
If I add the `--eval` flag to the init line, it's OK, but it was not needed previously.

Possible culprit: a79134e

### Theme

```yaml
# yaml-language-server: $schema=https://raw.githubusercontent.com/JanDeDobbeleer/oh-my-posh/main/themes/schema.json
version: 3
shell_integration: true
patch_pwsh_bleed: true
final_space: true
console_title_template: |-
  {{ if .Root -}}
    # {{- " " }}
  {{- end -}}
  {{ .UserName -}} @ {{- .HostName -}} :
  {{- if eq .PWD "~" -}}
    ~
  {{- else -}}
    {{ .Folder }}
  {{- end }}
palette:
  colorA: '#2f2f2f'
  colorB: '#3d3d3d'
blocks:
  # 1st line left
  - type: prompt
    alignment: left
    segments:
      - type: session
        style: powerline
        leading_powerline_symbol: 
        background: darkGray
        template: |-
          {{ if .Root -}}
            <red>
          {{- else if .SSHSession -}}
            <green>
          {{- else -}}
            <green>
          {{- end -}}
          {{ " " -}} <b> {{- .UserName -}} </b></>@ {{- .HostName }}
        cache: { strategy: session, duration: 168h }
      - type: os
        properties:
          windows: 
        style: powerline
        powerline_symbol: 
        foreground: black
        background: blue
        template: |-
          {{ if .WSL -}}
             {{- " " }}
          {{- end -}}
          {{ .Icon }} {{- " " }}
        cache: { strategy: session, duration: 168h }
      - type: path
        properties:
          style: agnoster_full
        style: powerline
        powerline_symbol: 
        foreground: black
        background: yellow
        template:  {{ path .Path .Location }}
      - type: git
        properties:
          fetch_stash_count: true
          fetch_status: true
          fetch_upstream_icon: true
          fetch_worktree_count: true
        style: powerline
        powerline_symbol: 
        foreground_templates:
          - |-
            {{ if or .Working.Changed .Staging.Changed -}}
              lightYellow
            {{- end }}
          - |-
            {{ if lt 0 .Behind -}}
              lightCyan
            {{- end }}
          - |-
            {{ if lt 0 .Ahead -}}
              lightMagenta
            {{- end }}
        background: p:colorA
        template: |-
          {{ .UpstreamIcon }} {{ .HEAD }} {{ .BranchStatus }}
          {{- if .Working.Changed -}}
            {{ " " -}}  <lightRed> {{- .Working.String -}} </>
          {{- end }}
          {{- if and .Working.Changed .Staging.Changed -}}
            {{ " " -}} |
          {{- end }}
          {{- if .Staging.Changed -}}
            {{ " " -}}  <lightGreen> {{- .Staging.String -}} </>
          {{- end }}
          {{- if lt 0 .StashCount -}}
            {{ " " -}}  {{ .StashCount }}
          {{- end }}
          {{- if lt 0 .WorktreeCount -}}
            {{ " " -}}  {{ .WorktreeCount }}
          {{- end }}
  # 1st line right
  - type: prompt
    alignment: right
    overflow: hide
    segments:
      - type: mvn
        properties:
          display_mode: always
          home_enabled: true
        style: accordion
        leading_powerline_symbol: 
        foreground: default
        background: p:colorA
        template: |-
          {{ if not .Error -}}
            <#f79a24></> mvn {{ url .Full .URL }}
          {{- else if ne .Error "NO VERSION" -}}
            {{ .Error }}
          {{- end }}
      - type: java
        properties:
          display_mode: always
          home_enabled: true
          version_url_template: |-
            {{ if lt 1 (.Major | int) -}}
              https://www.oracle.com/java/technologies/javase/ {{- .Full | replace "." "-" -}} -relnotes.html
            {{- else -}}
              https://www.oracle.com/java/technologies/javase/ {{- .Minor -}} u-relnotes.html
            {{- end }}
        style: accordion
        powerline_symbol: 
        foreground: default
        background: p:colorB
        template: |-
          {{ if not .Error -}}
            <#5283a2> </>
            {{- if .Env.JAVA_HOME -}}
              {{ path "jdk" .Env.JAVA_HOME }}
            {{- else -}}
              jdk
            {{- end -}}
            {{ " " -}} {{ url .Full .URL }}
          {{- else if ne .Error "NO VERSION" -}}
            {{ .Error }}
          {{- end }}
      - type: docker
        style: accordion
        powerline_symbol: 
        foreground: default
        background: p:colorA
        template: <#1d63ed></> docker[<lightRed> {{- .Context -}} </>]
      - type: node
        properties:
          display_mode: always
          home_enabled: true
        style: accordion
        powerline_symbol: 
        foreground: default
        background: p:colorB
        template: |-
          {{ if not .Error -}}
            <#539e43></> node {{ url .Full .URL }}
          {{- else if ne .Error "NO VERSION" -}}
            {{ .Error }}
          {{- end }}
      - type: python
        properties:
          display_mode: always
          home_enabled: true
        style: accordion
        powerline_symbol: 
        foreground: default
        background: p:colorA
        template: |-
          {{ if not .Error -}}
            <#f7c639></> py
          {{- if .Venv -}}
            [<lightGreen> {{- .Venv -}} </>]
          {{- end -}}
          {{ " " -}} {{ url .Full .URL }}
          {{- else if ne .Error "NO VERSION" -}}
            {{ .Error }}
          {{- end }}
      - type: upgrade
        properties:
          cache_duration: 12h
        style: powerline
        powerline_symbol: 
        foreground: default
        background: p:colorB
        template: <white></> omp {{ .Current }} <lightGreen> {{ .Latest -}} </>
      - type: shell
        style: powerline
        powerline_symbol: 
        foreground: black
        background: blue
        template:  {{ .Name }}
        cache: { strategy: session, duration: 168h }
      - type: time
        style: powerline
        powerline_symbol: 
        background: darkGray
        template:  {{ .CurrentDate | date .Format }}
  # 2nd line left
  - type: prompt
    newline: true
    alignment: left
    segments:
      - type: text
        style: plain
        template: |-
          {{ if .Root -}}
            <red>#
          {{- else -}}
            <green>$
          {{- end -}}
          </>
        cache: { strategy: session, duration: 168h }
  # 2nd line right
  - type: rprompt
    segments:
      - type: executiontime
        style: plain
        template: <red> </>  {{ .FormattedMs }} <red> </> {{- " " }}
      - type: status
        properties:
          always_enabled: true
        style: plain
        template: |-
           {{- " " }}
          {{- if .Error -}}
            <red>
          {{- else -}}
            <green>
          {{- end -}}
          </> {{ .String }}
```

### What OS are you seeing the problem on?

Windows

### Which shell are you using?

powershell

### Log output

```Shell
Version: 27.1.1

Shell: shell (7.5.3)

Prompt:

 geth   ~                                                                                                                                                                                                                                                   in shell at 11:00:41

Segments:

ConsoleTitle(true)                        -   0 ms
Session(true)                             -   0 ms
Path(true)                                -   1 ms
Git(false)                                -   0 ms
Root(false)                               -   0 ms
Status(true)                              -   0 ms
Node(false)                               -   0 ms
Go(false)                                 -   0 ms
Python(false)                             -   0 ms
Shell(true)                               -   0 ms
Time(true)                                -   0 ms

Run duration: 9.6508ms

Cache path: C:\Users\geth\AppData\Local\oh-my-posh

Config path: no --config set, using default built-in configuration

Logs:

[DEBUG] 11:00:41.248 debug.go:39 → logging enabled, raw mode: true
[DEBUG] 11:00:41.248 store.go:]:137 → (session) key not found: prompt_count_cache
[TRACE] 11:00:41.248 store.go:](session prompt_count_cache) - 0s
[TRACE] 11:00:41.248 terminal.go:setPromptCount() - 0s
[TRACE] 11:00:41.248 terminal.go:GOOS() - 0s
[DEBUG] 11:00:41.248 terminal.go:setPwd:102 → C:\Users\geth
[TRACE] 11:00:41.248 terminal.go:setPwd() - 0s
[TRACE] 11:00:41.248 terminal.go:Init() - 0s
[TRACE] 11:00:41.248 init.go:SessionID() - 0s
[DEBUG] 11:00:41.248 path_windows.go:PackageFamilyName:44 → failed to get PackageFamilyName
[TRACE] 11:00:41.248 path_windows.go:PackageFamilyName() - 0s
[TRACE] 11:00:41.248 path.go:Path() - 0s
[DEBUG] 11:00:41.255 store.go:init:94 → (session) loading prompt_count_cache
[DEBUG] 11:00:41.255 store.go:init:94 → (session) loading template_cache
[TRACE] 11:00:41.255 store.go:init(session pwsh.340d8abd-0750-47c0-9377-d825828e2fa1.omp.cache) - 6.8173ms
[TRACE] 11:00:41.255 path.go:Path() - 0s
[DEBUG] 11:00:41.255 store.go:init:94 → (device) loading upgrade_segment
[DEBUG] 11:00:41.255 store.go:init:94 → (device) loading INITVERSIONPWSH
[DEBUG] 11:00:41.255 store.go:init:94 → (device) loading DSC_CONFIGURATION
[DEBUG] 11:00:41.255 store.go:init:94 → (device) loading DSC_SHELL
[DEBUG] 11:00:41.255 store.go:init:94 → (device) loading https://raw.githubusercontent.com/g1bb0nz/g1bb0nz-etc-opt/main/g1bb0nz.omp.yaml.etag
[DEBUG] 11:00:41.255 store.go:init:94 → (device) loading https://raw.githubusercontent.com/g1bb0nz/g1bb0nz-etc-opt/main/g1bb0nz.omp.yaml.data
[TRACE] 11:00:41.255 store.go:init(device omp.cache) - 0s
[DEBUG] 11:00:41.255 store.go:]:137 → (device) key not found: reload
[TRACE] 11:00:41.255 store.go:](device reload) - 0s
[DEBUG] 11:00:41.256 store.go:]:137 → (session) key not found: CONFIG
[TRACE] 11:00:41.256 store.go:](session CONFIG) - 531µs
[DEBUG] 11:00:41.256 gob.go:Get:37 → no cached config found
[TRACE] 11:00:41.256 load.go:Load() - 0s
[TRACE] 11:00:41.256 gob.go:Get() - 531µs
[TRACE] 11:00:41.256 terminal.go:Shell() - 0s
[TRACE] 11:00:41.256 terminal.go:Flags() - 0s
[DEBUG] 11:00:41.256 store.go:]:150 → (session) found entry: template_cache - {map[Path:0xc0000c4480 Session:0xc00021f6b0 Shell:0xc0001b47d0 Status:0xc0001b4910 Time:0xc0001b48c0] map[] ~ geth  geth geth-desktop 7.5.3 pwsh C:\Users\geth windows 27.1.1 1 0 0 0 false false}
[TRACE] 11:00:41.256 store.go:](session template_cache) - 0s
[TRACE] 11:00:41.256 cache.go:restoreCache() - 0s
[DEBUG] 11:00:41.256 writer.go:Init:107 → terminal program: Windows Terminal
[DEBUG] 11:00:41.256 writer.go:Init:108 → terminal shell: shell
[DEBUG] 11:00:41.256 terminal.go:Getenv:64 → NO DATA
[TRACE] 11:00:41.256 terminal.go:Getenv(OMP_CACHE_DISABLED) - 0s
[DEBUG] 11:00:41.256 store.go:]:137 → (device) key not found: accent_color
[TRACE] 11:00:41.256 store.go:](device accent_color) - 0s
[DEBUG] 11:00:41.256 terminal_windows.go:WindowsRegistryKeyValue:198 → ColorizationColor(DWORD): 0xC4767676
[TRACE] 11:00:41.256 terminal_windows.go:WindowsRegistryKeyValue(HKEY_CURRENT_USER\Software\Microsoft\Windows\DWM\ColorizationColor) - 0s
[TRACE] 11:00:41.256 colors_windows.go:GetAccentColor() - 0s
[TRACE] 11:00:41.256 colors.go:SetAccentColor() - 0s
[TRACE] 11:00:41.256 terminal.go:Shell() - 0s
[DEBUG] 11:00:41.256 terminal.go:Getenv:64 → 7.5.3
[TRACE] 11:00:41.256 terminal.go:Getenv(POSH_SHELL_VERSION) - 0s
[DEBUG] 11:00:41.256 debug.go:PrintDebug:24 → segment: Title
[DEBUG] 11:00:41.256 text.go:patchTemplate:149 → {{ .Shell }} in {{ .Folder }}
[TRACE] 11:00:41.256 terminal.go:Flags() - 0s
[DEBUG] 11:00:41.256 terminal.go:Getenv:64 → 1
[TRACE] 11:00:41.256 terminal.go:Getenv(POSH_CURSOR_LINE) - 0s
[DEBUG] 11:00:41.256 terminal.go:Getenv:64 → 1
[TRACE] 11:00:41.256 terminal.go:Getenv(POSH_CURSOR_COLUMN) - 0s
[TRACE] 11:00:41.256 terminal.go:Flags() - 0s
[TRACE] 11:00:41.256 terminal.go:Flags() - 0s
[TRACE] 11:00:41.256 terminal.go:Flags() - 0s
[DEBUG] 11:00:41.256 home.go:12 → C:\Users\geth
[TRACE] 11:00:41.256 terminal.go:GOOS() - 0s
[TRACE] 11:00:41.256 terminal.go:Flags() - 0s
[DEBUG] 11:00:41.256 segment.go:Execute:109 → segment: Status
[DEBUG] 11:00:41.256 store.go:]:137 → (session) key not found: toggle_cache
[TRACE] 11:00:41.256 store.go:](session toggle_cache) - 0s
[DEBUG] 11:00:41.256 segment.go:isToggled:230 → no toggles found
[TRACE] 11:00:41.256 terminal.go:StatusCodes() - 0s
[DEBUG] 11:00:41.256 home.go:12 → C:\Users\geth
[TRACE] 11:00:41.256 terminal.go:GOOS() - 0s
[DEBUG] 11:00:41.256 segment.go:Execute:109 → segment: Path
[DEBUG] 11:00:41.256 properties.go:GetString:46 → status_template: {{ .Code }}
[DEBUG] 11:00:41.256 store.go:]:137 → (session) key not found: toggle_cache
[TRACE] 11:00:41.256 store.go:](session toggle_cache) - 0s
[DEBUG] 11:00:41.256 segment.go:isToggled:230 → no toggles found
[DEBUG] 11:00:41.256 properties.go:GetBool:40 → display_cygpath: false
[TRACE] 11:00:41.256 terminal.go:GOOS() - 0s
[TRACE] 11:00:41.256 separator.go:Separator() - 0s
[TRACE] 11:00:41.256 terminal.go:Shell() - 0s
[TRACE] 11:00:41.256 terminal.go:GOOS() - 0s
[TRACE] 11:00:41.256 terminal.go:Flags() - 0s
[DEBUG] 11:00:41.256 text.go:patchTemplate:149 → {{ .Code }}
[DEBUG] 11:00:41.256 properties.go:GetBool:40 → mapped_locations_enabled: true
[DEBUG] 11:00:41.256 home.go:12 → C:\Users\geth
[TRACE] 11:00:41.256 terminal.go:GOOS() - 0s
[DEBUG] 11:00:41.256 properties.go:GetString:46 → windows_registry_icon: 
[DEBUG] 11:00:41.256 segment.go:Execute:109 → segment: Git
[DEBUG] 11:00:41.256 properties.go:GetBool:40 → always_enabled: true
[DEBUG] 11:00:41.256 home.go:12 → C:\Users\geth
[TRACE] 11:00:41.256 separator.go:Separator() - 0s
[DEBUG] 11:00:41.256 store.go:]:137 → (session) key not found: toggle_cache
[TRACE] 11:00:41.256 terminal.go:Flags() - 0s
[TRACE] 11:00:41.256 store.go:](session toggle_cache) - 0s
[DEBUG] 11:00:41.256 segment.go:isToggled:230 → no toggles found
[DEBUG] 11:00:41.256 home.go:12 → C:\Users\geth
[TRACE] 11:00:41.256 terminal.go:GOOS() - 0s
[DEBUG] 11:00:41.256 segment.go:Execute:109 → segment: Root
[DEBUG] 11:00:41.256 store.go:]:137 → (session) key not found: toggle_cache
[DEBUG] 11:00:41.256 properties.go:GetBool:40 → disable_with_jj: false
[TRACE] 11:00:41.256 store.go:](session toggle_cache) - 0s
[DEBUG] 11:00:41.256 segment.go:isToggled:230 → no toggles found
[TRACE] 11:00:41.256 terminal.go:GOOS() - 0s
[DEBUG] 11:00:41.256 properties.go:GetString:46 → home_icon: ~
[DEBUG] 11:00:41.256 properties.go:GetKeyValueMap:64 → mapped_locations: map[]
[TRACE] 11:00:41.256 separator.go:Separator() - 0s
[TRACE] 11:00:41.256 terminal.go:GOOS() - 0s
[TRACE] 11:00:41.256 separator.go:Separator() - 0s
[TRACE] 11:00:41.256 terminal.go:GOOS() - 0s
[TRACE] 11:00:41.256 terminal_windows.go:Root() - 0s
[TRACE] 11:00:41.256 terminal.go:GOOS() - 0s
[TRACE] 11:00:41.256 terminal.go:Flags() - 0s
[TRACE] 11:00:41.256 terminal.go:GOOS() - 0s
[TRACE] 11:00:41.256 terminal.go:GOOS() - 0s
[DEBUG] 11:00:41.256 home.go:12 → C:\Users\geth
[TRACE] 11:00:41.256 terminal.go:GOOS() - 0s
[TRACE] 11:00:41.256 terminal.go:Flags() - 0s
[DEBUG] 11:00:41.256 segment.go:Execute:109 → segment: Session
[DEBUG] 11:00:41.256 properties.go:GetStringArray:70 → cycle: []
[DEBUG] 11:00:41.256 properties.go:GetString:46 → folder_separator_template:
[DEBUG] 11:00:41.256 store.go:]:137 → (session) key not found: toggle_cache
[DEBUG] 11:00:41.256 properties.go:GetString:46 → folder_separator_icon: \
[TRACE] 11:00:41.256 store.go:](session toggle_cache) - 0s
[DEBUG] 11:00:41.256 segment.go:isToggled:230 → no toggles found
[DEBUG] 11:00:41.256 properties.go:GetBool:40 → cycle_folder_separator: false
[DEBUG] 11:00:41.256 properties.go:GetString:46 → folder_format: %s
[DEBUG] 11:00:41.256 properties.go:GetString:46 → edge_format: %s
[DEBUG] 11:00:41.256 terminal.go:Getenv:64 → NO DATA
[DEBUG] 11:00:41.256 properties.go:GetString:46 → left_format: %s
[TRACE] 11:00:41.256 terminal.go:Getenv(SSH_CONNECTION) - 0s
[DEBUG] 11:00:41.256 properties.go:GetString:46 → right_format: %s
[DEBUG] 11:00:41.256 terminal.go:Getenv:64 → NO DATA
[TRACE] 11:00:41.256 terminal.go:Getenv(SSH_CLIENT) - 0s
[TRACE] 11:00:41.256 terminal.go:Flags() - 0s
[TRACE] 11:00:41.256 terminal.go:GOOS() - 0s
[TRACE] 11:00:41.256 terminal.go:StackCount() - 0s
[DEBUG] 11:00:41.256 text.go:patchTemplate:149 →  {{ if .Data.SSHSession }} {{ end }}{{ .UserName }}
[DEBUG] 11:00:41.256 terminal.go:ResolveSymlink:195 → C:\Users\geth
[TRACE] 11:00:41.256 terminal.go:ResolveSymlink(C:\Users\geth) - 0s
[ERROR] 11:00:41.256 terminal.go:HasParentFilePath:464 → GetFileAttributesEx .git: The system cannot find the file specified.
[TRACE] 11:00:41.256 terminal.go:HasParentFilePath(.git) - 0s
[DEBUG] 11:00:41.257 win32_windows.go:isWriteable:259 → not current user or in group
[DEBUG] 11:00:41.257 win32_windows.go:isWriteable:263 → current user is member of S-1-5-32-544
[DEBUG] 11:00:41.257 win32_windows.go:isWriteable:271 ↓
    WRITE_DAC
    WRITE_OWNER
    SYNCHRONIZE
    DELETE
    READ_CONTROL
[DEBUG] 11:00:41.257 win32_windows.go:isWriteable:273 → user has write access
[TRACE] 11:00:41.257 terminal_windows.go:DirIsWritable() - 567.1µs
[DEBUG] 11:00:41.257 text.go:patchTemplate:149 →   {{ path .Data.Path .Data.Location }}
[DEBUG] 11:00:41.257 text.go:patchTemplate:149 →  {{ if gt .Code 0 }}{{ else }}{{ end }}
[DEBUG] 11:00:41.257 text.go:patchTemplate:149 → {{ if gt .Code 0 }}p:red{{ end }}
[DEBUG] 11:00:41.257 text.go:patchTemplate:149 → {{ if gt .Code 0 }}p:red{{ end }}
[TRACE] 11:00:41.257 terminal.go:Shell() - 0s
[TRACE] 11:00:41.257 terminal.go:Flags() - 0s
[DEBUG] 11:00:41.257 home.go:12 → C:\Users\geth
[TRACE] 11:00:41.257 terminal.go:GOOS() - 0s
[DEBUG] 11:00:41.257 segment.go:Execute:109 → segment: Time
[DEBUG] 11:00:41.257 store.go:]:137 → (session) key not found: toggle_cache
[TRACE] 11:00:41.257 store.go:](session toggle_cache) - 0s
[DEBUG] 11:00:41.257 segment.go:isToggled:230 → no toggles found
[TRACE] 11:00:41.257 terminal.go:Flags() - 0s
[DEBUG] 11:00:41.257 properties.go:GetString:46 → time_format: 15:04:05
[TRACE] 11:00:41.257 terminal.go:Flags() - 0s
[DEBUG] 11:00:41.257 home.go:12 → C:\Users\geth
[DEBUG] 11:00:41.257 home.go:12 → C:\Users\geth
[TRACE] 11:00:41.257 terminal.go:GOOS() - 0s
[TRACE] 11:00:41.257 terminal.go:GOOS() - 0s
[DEBUG] 11:00:41.257 segment.go:Execute:109 → segment: Python
[DEBUG] 11:00:41.257 segment.go:Execute:109 → segment: Go
[DEBUG] 11:00:41.257 store.go:]:137 → (session) key not found: toggle_cache
[DEBUG] 11:00:41.257 store.go:]:137 → (session) key not found: toggle_cache
[TRACE] 11:00:41.257 store.go:](session toggle_cache) - 0s
[TRACE] 11:00:41.257 store.go:](session toggle_cache) - 0s
[DEBUG] 11:00:41.257 segment.go:isToggled:230 → no toggles found
[DEBUG] 11:00:41.257 segment.go:isToggled:230 → no toggles found
[DEBUG] 11:00:41.257 properties.go:GetString:46 → display_mode: files
[TRACE] 11:00:41.257 terminal.go:Flags() - 0s
[DEBUG] 11:00:41.257 properties.go:GetStringArray:70 → extensions: [*.go go.mod go.sum go.work go.work.sum]
[DEBUG] 11:00:41.257 properties.go:GetStringArray:70 → extensions: [*.py *.ipynb pyproject.toml venv.bak]
[DEBUG] 11:00:41.257 properties.go:GetStringArray:70 → folders: []
[DEBUG] 11:00:41.257 properties.go:GetStringArray:70 → folders: [.venv venv virtualenv venv-win pyenv-win]
[DEBUG] 11:00:41.257 properties.go:GetBool:40 → home_enabled: false
[DEBUG] 11:00:41.257 properties.go:GetBool:40 → home_enabled: false
[DEBUG] 11:00:41.257 home.go:12 → C:\Users\geth
[DEBUG] 11:00:41.257 home.go:12 → C:\Users\geth
[DEBUG] 11:00:41.257 home.go:12 → C:\Users\geth
[TRACE] 11:00:41.257 terminal.go:GOOS() - 0s
[TRACE] 11:00:41.257 terminal.go:Flags() - 0s
[DEBUG] 11:00:41.257 segment.go:Execute:109 → segment: Shell
[DEBUG] 11:00:41.257 store.go:]:137 → (session) key not found: toggle_cache
[DEBUG] 11:00:41.257 home.go:12 → C:\Users\geth
[TRACE] 11:00:41.257 store.go:](session toggle_cache) - 0s
[DEBUG] 11:00:41.257 segment.go:isToggled:230 → no toggles found
[DEBUG] 11:00:41.257 segment.go:Execute:109 → segment: Node
[DEBUG] 11:00:41.257 properties.go:GetKeyValueMap:64 → mapped_shell_names: map[]
[TRACE] 11:00:41.257 terminal.go:Shell() - 0s
[TRACE] 11:00:41.257 terminal.go:Flags() - 0s
[DEBUG] 11:00:41.257 store.go:]:137 → (session) key not found: toggle_cache
[TRACE] 11:00:41.257 store.go:](session toggle_cache) - 0s
[DEBUG] 11:00:41.257 segment.go:isToggled:230 → no toggles found
[DEBUG] 11:00:41.257 properties.go:GetStringArray:70 → extensions: [*.js *.ts package.json .nvmrc pnpm-workspace.yaml .pnpmfile.cjs .vue]
[DEBUG] 11:00:41.257 properties.go:GetStringArray:70 → folders: []
[DEBUG] 11:00:41.257 properties.go:GetBool:40 → home_enabled: false
[DEBUG] 11:00:41.257 home.go:12 → C:\Users\geth
[DEBUG] 11:00:41.257 text.go:patchTemplate:149 → in <p:blue><b>{{ .Data.Name }}</b></>
[DEBUG] 11:00:41.257 text.go:patchTemplate:149 → at <p:blue><b>{{ .Data.CurrentDate | date "15:04:05" }}</b></>
[TRACE] 11:00:41.257 terminal.go:Shell() - 0s
[TRACE] 11:00:41.257 terminal.go:Flags() - 0s
[TRACE] 11:00:41.257 terminal.go:Shell() - 0s
[DEBUG] 11:00:41.257 terminal_windows.go:TerminalWidth:99 → terminal width: 280
[TRACE] 11:00:41.257 terminal_windows.go:TerminalWidth() - 0s
[TRACE] 11:00:41.257 path.go:Path() - 0s
```

--- a/src/cli/init.go
+++ b/src/cli/init.go
@@ -167,7 +167,7 @@ func getFullCommand(cmd *cobra.Command, args []string) string {
 }
 
 func initCache(sh string) {
-	if !printOutput && (sh == shell.PWSH || sh == shell.ELVISH) {
+	if !printOutput && eval && (sh == shell.PWSH || sh == shell.ELVISH) {
 		cache.Init(sh)
 		return
 	}

--- a/src/shell/init.go
+++ b/src/shell/init.go
@@ -55,9 +55,11 @@ func Init(env runtime.Environment, feats Features) string {
 		}
 
 		var additionalParams string
+
 		if env.Flags().Strict {
 			additionalParams += " --strict"
 		}
+
 		if env.Flags().Eval {
 			additionalParams += " --eval"
 		}

